packages/yarnpkg-pnp/sources/loader/module.d.ts

```diff
@@ -10,6 +10,6 @@ declare module "module" {
   function _load(request: string, parent: NodeModule | null | undefined, isMain: boolean): any;
 
   interface Module extends NodeModule {
-    pnpApiPath?: import('@yarnpkg/fslib').PortablePath;
+    pnpApiPath?: import('@yarnpkg/fslib').PortablePath | null;
```

Comment:
https://github.com/yarnpkg/berry/blob/73ed9b738e7755936aa62a92dd86a368e0c8bab0/packages/yarnpkg-pnp/sources/loader/makeManager.ts#L102
Since we check for not null here I updated the type to reflect that it can be null